sabre/dav/.travis.yml

```diff
@@ -8,17 +8,22 @@ php:
 matrix:
   fast_finish: true
 
+env:
```

Comment:
True, will remove. Also I see that the bin folder has sneaked in back, will remove it too.